indicadores/models.py

```diff
@@ -1,3 +1,158 @@
+    multiplicador: DecimalField[12,10]
+        This is the actual value that is used in calculations. It's
+        purpose is to be able to calculate in the case of incomes and
+        expenses how much the monto represents per month, by multiplying
+        the income by this number.
+    """
+
+    periodicidad = models.TextField()
+    multiplicador = models.DecimalField(max_digits=12, decimal_places=10)
+
+    def __str__(self):
+        """ This returns the value or the periodicidad attribute
+
+        """
+        return '{}'.format(self.periodicidad)
+
+
+class Transaccion(models.Model):
+    """ This model details the income and expenses of a family has.
+
+    Incomes are very important as they are used to directly
+    calculate the scholarship of a student, and are an essential
+    part of the calulation of many of the indicators that are going
+    to be displayed.
+
+    Attributes:
+    -----------
+    familia : ForeignKey
+        This field matches a transaction with a family.
+    activo : boolean
+        This field indicates whether a certain transaction is currently
+        being transacted by the family.
+    monto : DecimalField[12,2]
+        This field stores the acutal monetary amount that gets exchanged.
+        The currency of this amount is MXN.
+    periodicidad : ForeignKey
+        This field references the value of how often the mone is exchanged
+        by the family.
+    observacion : Text
+        This field stores any additional comments that clarify something
+        about the transaction.
+    es_ingreso : BooleanField
+        This field indicates whether a certain transaction is an income
+        or an expense.
+    """
+    familia = models.ForeignKey(Familia)
+    activo = models.BooleanField(default=True)
+    monto = models.DecimalField(max_digits=12, decimal_places=2)
+    periodicidad = models.ForeignKey(Periodo)
+    observacion = models.TextField()
+    es_ingreso = models.BooleanField()
+
+    def obtener_valor_de_transaccion(self):
+        """ If a transaction is an expense, returns a negative value
+
+        This method gives you the value of a transaction, and is the
+        correct way of getting this value.
+        """
+        valor_transaccion = self.monto
+        if self.es_ingreso is not True:
+            valor_transaccion = valor_transaccion * decimal.Decimal(-1.0)
+        if self.activo is True:
+            return valor_transaccion
+        else:
+            return 0.0
+
+    def obtener_valor_mensual(self):
+        """ Calculates how much money is exchanged in a month.
+
+        """
+        return self.obtener_valor_de_transaccion() * self.periodicidad.multiplicador
+
+    def __str__(self):
+        """ Returns the calculated mensual transaction, formatted as money.
+
+        """
+        signo_opcional = ''
+        valor_mensual_transaccion = self.obtener_valor_mensual()
+        if valor_mensual_transaccion < 0.0:
```

Comment:
I guess it's safer to make the comparison as decimal.Decimal('0.0'). And in general, use literals in the constructors where we're using them.